students/gyuchan/users/views.py

```diff
@@ -47,10 +49,12 @@ def post(self, request):
 
             user = User.objects.get(email=email)
 
-            if user.password != password:
+            if not bcrypt.checkpw(password.encode('utf-8'), user.password.encode('utf-8')):
                 return JsonResponse({'message': 'INVALID_USER'}, status=401)
+            
+            access_token = jwt.encode({'id': user.id}, SECRET_KEY, ALGORITHM)
 
-            return JsonResponse({'message': 'SUCCESS'}, status=200)
+            return JsonResponse({'token': access_token}, status=200)
```

Comment:
```suggestion
            return JsonResponse({'access_token': access_token}, status=200)
```
조금 더 명확한 변수명으로 작성해주세요~ 토큰은 다른 토큰도 있을 수 있으니까요!